### DISCUSSION TITLE
11 particular (and not) suggestions: they're my best ones (or not). Well,read them...

### DISCUSSION THREAD

**Discussion by ghost** [voxeltycoon/issues#550] [General]
Suggestion 1 [wagons & cargo]:
in the column used to choose what wagons loading or unloading, those same wagons should be numbered starting from the first one and not from locomotives & tenders. It causes me some problems when i'm calculating. 
I also wish to see the cargo of each car (as in the details page). In fact I have problems to recognize two cars of the same type if they are loaded with two different goods.

Suggestion 2 [electro shock]:
There is a button to decide on which side of tracks to place railway poles. great, but it works only before placing rails.
Is it possible having another same button to use for placing those poles on tracks already built before?

Suggestion 3 [finish him]:
I'd need a button to delete all destination of a truck or train at once.

Suggestion 4 [sooo soft]:
Two words: smoother curves. Actually one piece of curve occupies 4x3 of area; I'm proposing longer and wider turns, for example 8x6. They would be suitable for high-speed railway lines.

Suggestion 5 [undesirable break]:
Replace the "go to" key with the "pass to" command (a train that stops in the middle of a high-speed line just because it has to pass there creates problems for the trains behind it).

Suggestion 6 [who passes?]:
Several times I have seen a train stopped at the station to load/unload and behind it another train arriving at high speed. the problem is the mechanics: if the train leaves the station and passes its signal immediately before the other train passes its own, the latter stops suddenly to let it pass. It's not realistic and I know it is not a bug. For this reason I suggest you to do this: in this situation you have to calculate the braking time of the fastest train and understand if it has enough space to stop. if the answer is no, the train at the station waits, but if there is enough space, the train can safely leave the station.
Moreover, a smaller space than would be used under normal conditions can be set as the minimum braking distance. in such a way as to be more flexible in certain complicated situations, such as in a freight terminal. the important thing is that the reduction of the braking distance is not excessive and that allows the train to realize a realistic braking.
-(video in Discord "Suggestion" channel, 19/07/2019

Suggestion 7 [drawing lines]:
Could you add the possibility of directly tracing a line on the route that I would like a train to travel?
In fact I have some problems with trains that travel in a haywire network. Sometimes I set certain destinations in such a way that the trains follow certain paths, but it seems that they don't care at all and they travel along other longer routes.

Suggestion 8 [more and more horns]:
I think it's already planned, but could you add more than a single horn for all the locomotives? I mean that every type of locomotive (class 66, 82 class, gs-4 ...) should have a custom horn that does not always repeat in the same way but has at least one variant.
-(I already did it in private. If you are interested, you can tell me)-

Suggestion 9 [flying stations]:
I was thinking about elevated railway stations, like those of the subway in New York. maybe it could be a normal station built on steel pillars or above an arc bridge.
At this point I propose you to make the same also for normal rails and not only stations.

Suggestion 10 [you shall not... turn]:
About the "turn-around" button of the trains, maybe it would be more convenient if instead of being a generalized function it was a button to be activated in the train schedule, identified by an icon next to the name of each station.

Suggestion 11 [Safari]:
Try to add a new section in the main menu where gamers can share their own maps or play in the other people's ones (worlds possibly made not for playing but for exploring them, like scale models of reality).
That will become the new "Safari" section.

Thank you!

**Comment by katepolak** [comment]
Suggestion 2 is already possible by clicking the segment again after building poles using the electrification tool, first click will switch the side and second one will remove them

**Comment by ghost** [comment]
I know it. I was asking for another way to do that. Several times I was forced to spend twice a huge sum of money to change the side of poles, and I would like to avoid it.
So I asked for another button.

Inviato dal mio dispositivo Huawei

-------- Messaggio originale --------
Oggetto: Re: [voxeltycoon/suggestions] 11 particular (and not) suggestions: they're my best ones (or not). Well,read them... (#68)
Da: Jan Polák
A: voxeltycoon/suggestions
CC: Federico57 ,Author


Suggestion 2 is already possible by clicking the segment again after building poles using the electrification tool, first click will switch the side and second one will remove them
